WordPress/src/main/java/org/wordpress/android/ui/stories/intro/StoriesIntroDialogFragment.kt

```diff
@@ -50,14 +48,12 @@ class StoriesIntroDialogFragment : DialogFragment() {
         viewModel = ViewModelProvider(this, viewModelFactory)
                 .get(StoriesIntroViewModel::class.java)
 
-        if (VERSION.SDK_INT >= VERSION_CODES.M) {
-            val window: Window? = dialog.window
-            window?.let {
-                window.statusBarColor = dialog.context.getColorFromAttribute(attr.colorSurface)
-                if (!resources.configuration.isDarkTheme()) {
-                    window.decorView.systemUiVisibility = window.decorView
-                            .systemUiVisibility or View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
-                }
+        val window: Window? = dialog.window
+        window?.let {
+            window.statusBarColor = dialog.context.getColorFromAttribute(attr.colorSurface)
+            if (!resources.configuration.isDarkTheme()) {
+                window.decorView.systemUiVisibility = window.decorView
+                        .systemUiVisibility or View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
```

Comment:
🤔 why is this indented 8 spaces? Is it because it's part of the right operand of an assignment?

Reply:
🤷